virttest/qemu_vm.py

```diff
@@ -2145,6 +2149,7 @@ def sort_key(dev):
                                       params=net_params, cmdline_nd=cmd_nd))
             else:
                 device_driver = nic_params.get("device_driver", "pci-assign")
+                failover_pair_id = nic_params.get("failover_pair_id")
```

Comment:
It is better to move this line to add_pcidevice, but add_pcidevice is only used for network currently, so ACK.

Reply:
It must have to move into `add_pcidevice()`, otherwise,  add_pcidevice() can't use `failover_pair_id` in L883 directly.

Reply:
Seems I was wrong, it can work if not move.